test/jdk/javax/swing/JFileChooser/8194044/FileSystemRootTest.java

```diff
@@ -36,6 +36,10 @@
 
 public class FileSystemRootTest {
     public static void main(String[] args) throws Exception {
+        if (System.getProperty("os.name").equalsIgnoreCase("Windows 11")) {
```

Comment:
I guess the same behavior applies to versions newer than 11, right? If that's the case, perhaps you could check whether `getParentDirectory("C:\...\Documents")` returns `C:\..\Desktop`.

Reply:
I think we should assume it does apply to newer versions.
However that is moot, as I don't see much point in continuing to run on the old, nearly out of support Windows 10.
Additionally I strongly suspect that you'll find that Windows Server 2022 and definitely 2025 will have the same behavior and you aren't preventing the test running there.

And if this is fixed, the bug tracking the problem is gone. So you'd just have to file another one and we've artificially hidden the test failure.

So I think this PR should be withdrawn.